backend/src/services/ai/embedding.service.ts

```
@@ -2,7 +2,7 @@ import { OpenRouterProvider } from '@/providers/ai/openrouter.provider.js';
 import type { EmbeddingsRequest, EmbeddingsResponse } from '@insforge/shared-schemas';
 import logger from '@/utils/logger.js';
 import { AppError } from '@/api/middlewares/error.js';
-import { ERROR_CODES } from '@/types/error-constants.js';
+import { ERROR_CODES } from '@insforge/shared-schemas';
```

ESLint

[error] '@insforge/shared-schemas' import is duplicated.

(no-duplicate-imports)